core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.html

```diff
@@ -78,6 +78,25 @@ <h3>Regenerate contribution opportunities for a topic:</h3>
   </div>
 </mat-card>
 
+<mat-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Rollback exploration to latest known safe state:</h3>
```

Comment:
Maybe add an explanation in italics to explain what a "safe" state is.

Reply:
Done.

Reply:
Minor wording change: "Here, a "safe state" refers to the latest version of the exploration for which both snapshot content and snapshot metadata models also exist."

Reply:
Sure, done.